package.json

```diff
@@ -11,11 +11,16 @@
   "bin": {
     "pagespeed-score": "cli.js"
   },
+  "xo": {
+    "space": true,
+    "env":["node"]
```

Comment:
@sindresorhus On documentation the default value is really `false`. I believe we can keep it =]